docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+#. **Locate and create an inventory of all your hardware.**
+   - *Journalist Workstation* USBs
+   - *Admin Workstation* USB(s)
+   - *Secure Viewing Station* USB
+   - *Secure Viewing Station* computer
+   - Transfer and Export devices (USBs, optical drives, or external drives)
+   - Backup USBs/other storage media
+   - Servers
+   - Firewall
+
+   .. note:: The recommended SecureDrop setup includes only one *Secure Viewing
+      Station* USB. However, if you have been working remotely or
+      have a non-standard setup, you may have more than one *SVS USB*. It is
+      important that you locate all of these USBs, since they hold the most
+      sensitive data.
+#. **Decide whether to save a backup.**
+   If you want a backup, you can, follow our `backup guidelines`_. Once the
+   backup has been created, you can move it off of the *Admin Workstation* USB
+   and onto an encrypted device, such as a LUKS-encrypted drive. You will also
+   require the *SVS* USB and/or a backup of the SecureDrop Application Key.
+#. **Disconnect the firewall and the servers from the internet.**
+   Be sure to inform your network administrator of any changes to devices on
+   your network.
+#. **Wipe or destroy the USB drives.**
+   Because the USB drives used for SecureDrop are all LUKS-encrypted,
+   reformatting the USB drives (in particular, overwriting a portion of internal
+   storage called the **LUKS header**) should be sufficient to make any existing
+   data on those drives unrecoverable.
+
+   For example, you could use your *primary Tails USB* to launch Gnome Disks,
+   insert and identify the USB drive you are trying to erase, and reformat this
+   drive with a new, LUKS-encrypted partition, erasing the existing partition
+   data.
+
+   .. caution:: Be **very** sure you are reformatting the right drive.
+      You may want to use the Secure Viewing Station laptop for this procedure
+      to reduce the risk of accidentally erasing a drive on your regular-use
+      machine.
+
+   You may also choose to destroy the drives by physical means, such as using a
+   hammer or purpose-built shredder to pulverize or destroy the drive.
+#. **Wipe or destroy the storage drives on the servers.**
+   SecureDrop submissions are stored GPG-encrypted on the Application server.
+   Unless your SecureDrop Application Key is compromised (or a significant
+   vulnerability in GPG is discovered), access to the servers does not guarantee
+   access to the submissions and messages you have received.
+
+   That said, there is still some sensitive information on the servers,
+   including system logs and the SecureDrop database, which would yield
+   information on the number of submissions and replies stored on the server.
+
+   Physically destroying or encrypting the storage drives on the servers are the
+   best ways to ensure that data on the drives cannot be recovered.
+
+   Physically destroying SSD drives is not as straightforward as destroying
```

Comment:
It could be interesting to have external references to these guidelines. Similar to USB flash drives, I was under the impression that physically destroying the NAND chips and DRAM chips here would be sufficient.

For example NIST-SP800-88 [1] recommends, for both SSDs and spinning platters, which is in-line with your recommendations for the USB drives above:
```
Shred, Disintegrate, Pulverize, or Incinerate by burning the device in a licensed incinerator.
```

[1] : https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-88r1.pdf

Reply:
Yes, you're right--but the reason that I said it was 'not as straightforward' is that people might not have enough understanding to identify and destroy those specific chips, and might assume that the drive is inoperable when it is actually not (vs for example drilling holes through random parts of an HDD), so if they're opting for physical destruction they should do some research. This may seem overly paranoid, but it might not be considering we advise against FDE on the servers.  

I can be a bit more specific here though.

Reply:
See f38127f